.github/workflows/ci.yml

```diff
@@ -14,6 +14,18 @@ jobs:
   pest:
     name: Tests (Pest)
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [8.0]
+        laravel: [9.*, 8.*]
+        dependency-version: [prefer-stable]
```

Comment:
Things that only have a single value don't need to be in the matrix. If they're needed, add them to the env.

Matrix is for things that have multiple values and become combined with each other